flake.nix

```
@@ -1,5 +1,10 @@
 {
-  description = "A garnix module for postgreSQL";
+  description = ''A garnix module for projects using PostgreSQL.
+
+  Note: enabling PostgreSQL will automatically mark the deployed server as [persistent](https://garnix.io/docs/hosting/persistence).
+
+  [Documentation](https://garnix.io/docs/modules/postgresql) - [Source](https://github.com/garnix-io/postgresql-module).
+  '';
```

```suggestion
  description = ''
    A garnix module for projects using PostgreSQL.

    Note: enabling PostgreSQL will automatically mark the deployed server as [persistent](https://garnix.io/docs/hosting/persistence).

    [Documentation](https://garnix.io/docs/modules/postgresql) - [Source](https://github.com/garnix-io/postgresql-module).
  '';
```

Maybe this is nicer to read? (`nix` removes the leading newline.)